--- xdrfile-1.1.4/include/xdrfile.h	2014-04-17 23:17:03.248063551 +0200
+++ include/xdrfile/xdrfile.h	2014-04-17 23:17:03.244063551 +0200
@@ -102,7 +102,7 @@
 		   exdrINT, exdrFLOAT, exdrUINT, exdr3DX, exdrCLOSE, exdrMAGIC,
 		   exdrNOMEM, exdrENDOFFILE, exdrFILENOTFOUND, exdrNR };
 
-	extern char *exdr_message[exdrNR];
+/*	extern char *exdr_message[exdrNR]; */
 
 #define DIM 3
 	typedef float matrix[DIM][DIM];
--- xdrfile-1.1.4/include/xdrfile_xtc.h	2014-04-17 23:17:03.248063551 +0200
+++ include/xdrfile/xdrfile_xtc.h	2014-04-17 23:17:03.244063551 +0200
@@ -38,6 +38,7 @@
 extern "C" {
 #endif
 
+#include "xdrfile_trr.h"
 #include "xdrfile.h"
   
   /* All functions return exdrOK if succesfull. 
